src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -18,22 +18,9 @@
 
     public static final String MESSAGE_SUCCESS = "New task added: %1$s";
 
-    private final Task toAdd;
-
-    /**
-     * Convenience constructor using raw values.
-     *
-     * @throws IllegalValueException if any of the raw values are invalid
-     */
-    public AddCommand(String name) throws IllegalValueException {
-        this.toAdd = new Task(new Name(name));
-    }
-
-    @Override
-    public CommandResult execute() {
-        assert model != null;
-        model.addTask(toAdd);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
-    }
-
+	@Override
+	public CommandResult execute() {
+		// TODO Auto-generated method stub
+		return null;
+	}
```

Comment:
Right, but you need to do more than just blind removal of code. You need to think about why the code you are removing even exists in the first place.

Q: Why does `Parser` depend on `AddCommand.COMMAND_WORD`?

A: Because it parses the `add` command, which then returns an instance of `AddCommand`.

Q: What happens when the new instance of `AddCommand` is returned?

A: The `LogicManager` class calls `execute()` on it.

Q: With the changes in this PR, what happens when `execute()` is called?

A: `null` is returned.

But does the `LogicManager` even expect `null` to be returned? What happens if it gets a `null`? ......